src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+- [Device Disconnect](https://googlechrome.github.io/samples/web-bluetooth/device-disconnect.html) - disconnect and get notified from a disconnection of a BLE Device after connecting to it.
+- [Get Characteristics](https://googlechrome.github.io/samples/web-bluetooth/get-characteristics.html) - get all characteristics of an advertised service from a BLE Device.
+- [Get Descriptors](https://googlechrome.github.io/samples/web-bluetooth/get-descriptors.html) - get all characteristics' descriptors of an advertised service from a BLE Device.
+
+### Combining multiple operations
+
+- [GAP Characteristics](https://googlechrome.github.io/samples/web-bluetooth/gap-characteristics.html) - get all GAP characteristics of a BLE Device.
+- [Device Information Characteristics](https://googlechrome.github.io/samples/web-bluetooth/device-information-characteristics.html) - get all Device Information characteristics of a BLE Device.
+- [Link Loss](https://googlechrome.github.io/samples/web-bluetooth/link-loss.html) - set the Alert Level characteristic of a BLE Device (readValue & writeValue).
+- [Discover Services & Characteristics](https://googlechrome.github.io/samples/web-bluetooth/discover-services-and-characteristics.html) - discover all accessible primary services and their characteristics from a BLE Device.
+- [Automatic Reconnect](https://googlechrome.github.io/samples/web-bluetooth/automatic-reconnect.html) - reconnect to a disconnected BLE device using an exponential backoff algorithm.
+- [Read Characteristic Value Changed](https://googlechrome.github.io/samples/web-bluetooth/read-characteristic-value-changed.html) - read battery level and be notified of changes from a BLE Device.
+- [Read Descriptors](https://googlechrome.github.io/samples/web-bluetooth/read-descriptors.html) - read all characteristic's descriptors of a service from a BLE Device.
+- [Write Descriptor](https://googlechrome.github.io/samples/web-bluetooth/write-descriptor.html) - write to the descriptor "Characteristic User Description" on a BLE Device.
+
+Check out our [curated Web Bluetooth Demos] and [official Web Bluetooth Codelabs] as well.
+
+## Libraries
+
+- [web-bluetooth-utils] is a npm module that adds some convenience functions to
+  the API.
+- A Web Bluetooth API shim is available in [noble], the most popular Node.js BLE
+  central module. This allows you to webpack/browserify noble without the need
+  for a WebSocket server or other plugins.
+- [angular-web-bluetooth] is a module for [Angular] that abstracts away all the
+  boilerplate needed to configure the Web Bluetooth API.
+- [&lt;platinum-bluetooth>] is a set of [Polymer] elements to discover and
+  communicate with nearby Bluetooth devices based on the Web Bluetooth API. For
+  instance, here's how to read battery level from a nearby bluetooth device
+  advertising a Battery service:
```

Comment:
No longer maintained, right?

Reply:
This one has been archived indeed.
I was thinking about keeping it for the posterity but it looks like you suggest I should remove it. So let's do that ;)